404.html

```diff
@@ -15,12 +15,12 @@
 <!--Fallback page, don't allow indexing-->
 <meta name="robots" content="noindex">
 <h1>An open source P2P digital currency</h1>
+{% comment %}
 <p style="font-size:125%;">Bitcoin is a currency, a protocol, and a software that enables
 <ul style="font-size:125%">
-  <li>Instant peer to peer transactions</li>
```

Comment:
"peer to peer" implied "from a stranger" to me because when I use other P2P protocols like BitTorrent, most of my peers are strangers.

We could rephrase to "Instant transactions from friends", but that sounds like weak marketing to me.  It implies (correctly) that transactions from strangers aren't instant, so it comes off as a negative point.

I think it's better to remove it and replace it entirely with a completely different advantage of Bitcoin.